saviSimulation/savi/jason_processing/FieldAntenna.java

```diff
@@ -15,36 +15,27 @@
 	double wifi = 100;
 	//TODO: add list of messages here, then implement message delivery
 	
-	public FieldAntenna(int id, PVector position, SAVIWorld_model sim, int size, PShape image) {
-		
-		//int id, PVector pos, int pixels, String Type, SAVIWorld_model sim, PShape image
+	public FieldAntenna(int id, PVector position, SAVIWorld_model sim, int size, PShape image, double wifiProbWorking, int seed) {
 		
 		super(id, position, size, "FieldAntenna", sim, image);
-		
 		this.position = position;
```

Comment:
just noticed an issue here (perhaps introduced by me in an earlier commit): the attribute position is duplicated between this class and WorldObject (need to be removed from here)